sopel/bot.py

```diff
@@ -326,12 +340,13 @@ def remove_plugin(self, plugin, callables, jobs, shutdowns, urls):
                     regex = func.url_regex
                     if func == self.memory['url_callbacks'].get(regex):
                         self.unregister_url_callback(regex)
+                        LOGGER.debug('URL Callback unregistered %r', regex)
```

Comment:
```suggestion
                        LOGGER.debug('URL Callback unregistered: %r', regex)
```

Also, it's outside the range of lines that GitHub will let me add notes on, but the `self.unregister(*args)` loop above would be another good place for debug output.

Reply:
Looking at the current patch, this colon is still missing somehow.